admin/pre_reg/utils.py

```diff
@@ -5,10 +5,14 @@
 
 
 def get_prereg_reviewers():
-    return MyUser.objects.filter(
-        groups__name='prereg_group'
-    ).annotate(
-        fuller_name=Concat('first_name', Value(' '), 'last_name')
-    ).values_list(
-        'email', 'fuller_name'
-    )
+    # Note - fixes django.db.utils.OperationalError: no such table' error if ever one erases a db table and tries to remigrate
+    try:
+        return MyUser.objects.filter(
+            groups__name='prereg_group'
+        ).annotate(
+            fuller_name=Concat('first_name', Value(' '), 'last_name')
+        ).values_list(
+            'email', 'fuller_name'
+        )
+    except Exception:
```

Comment:
@zkraime It depends on what exceptions you're trying to account for. django ORM's filter is pretty forgiving. If your query doesn't return any results you should just get an empty queryset with no exception. If your `Group` model didn't have a `name` you would get a [FieldError](https://docs.djangoproject.com/en/1.9/ref/exceptions/#fielderror). Same thing if `email` or `fuller_name` weren't fields. 

Generally, when I'm putting in a try except I do it because I got an exception while functional testing and it was an exception I knew *could* happen to a user but I wanted to handle that in some way in order to explain it to them and not just give them an error without recourse. Chances are if you're querying on fields that don't exist, errything broke and messaging to the user isn't going to fix it. If you're trying to account for invalid user input, that should probably be done at the form.